PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -1317,6 +1317,21 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			return TRUE;
 		}
 
+		case NPPM_CLOSEDOC:
+		{
+			// Close a document without switching to it
+			int whichView = ((wParam != MAIN_VIEW) && (wParam != SUB_VIEW)) ? currentView() : static_cast<int32_t>(wParam);

```

Comment:
Just curious: why should `(wParam != MAIN_VIEW) && (wParam != SUB_VIEW)` be an option?

Thank you.

Reply:
It's copy pasted directly from `NPPM_ACTIVATEDOC` message above.

Reply:
Thank you. 👍

Reply:
@Yaron10 
> Just curious: why should (wParam != MAIN_VIEW) && (wParam != SUB_VIEW) be an option?

I guess (I coded long time ago) it's for preventing wParam from other values than MAIN_VIEW or SUB_VIEW.

Reply:
@donho,

Being extra safe is good. :)
Thank you. 👍